mathjax3-ts/output/chtml/Wrappers/mtable.ts

```diff
@@ -71,6 +72,58 @@ export class CHTMLmtable extends CHTMLWrapper {
         //
         this.numCols = this.childNodes.map(row => (row as CHTMLmtr).numCells).reduce((a, b) => Math.max(a, b));
         this.numRows = this.childNodes.length;
+        //
+        // Stretch the columns (rows are already taken care of in the CHTMLmtr wrapper)
+        //
+        for (let i = 0; i < this.numCols; i++) {
+            this.stretchColumn(i);
+        }
+    }
+
+    /*
+     * Handle horizontal stretching within the ith column
+     */
+    protected stretchColumn(i: number) {
+        let stretchy: CHTMLWrapper[] = [];
+        //
+        //  Locate and count the stretchy children
+        //
+        for (const row of (this.childNodes as CHTMLmtr[])) {
+            const cell = row.childNodes[i + row.firstCell];
+            if (cell) {
+                const child = cell.childNodes[0];
+                if (child.stretch.dir === DIRECTION.None && child.canStretch(DIRECTION.Horizontal)) {
+                    stretchy.push(child);
+                }
+            }
+        }
+        let count = stretchy.length;
+        let nodeCount = this.childNodes.length;
+        if (count && nodeCount > 1) {
+            let W = 0;
+            //
+            //  If all the children are stretchy, find the largest one,
+            //  otherwise, find the width of the non-stretchy children.
+            //
+            let all = (count > 1 && count === nodeCount);
```

Comment:
Why the parentheses?

Reply:
I have always preferred an assignment to a boolean operation to make it clear that the booleans all get done first (mixing `=` with things like `==` and so on is a potential confusion).  We talked about this once, and you agreed that it was OK to do that (and we could make it a style recommendation).